### ISSUE TITLE
Too many features have not been done

### ISSUE THREAD

**Issue by gguoss** [paritytech/parity-bitcoin#522]
Is this project not advanced?

**Comment by gguoss** [comment]
https://github.com/chainx-org/bitcoin-relay --- The github inherits from paritytech/parity-bitcoin and implements miner, transaction management and simple bitcoin wallet.

**Comment by gguoss** [comment]
Of course, we are preparing to do a rust version of bitcoin spv and implant into the substrate framework.

**Comment by 5chdn** [comment]
You could just submit your contributions as PR. :)

**Comment by thijstriemstra** [comment]
> maintenance-only mode. We are planning to add several missing features and optimisations

that's a contradiction. so what is it?
